components/Navbar.vue

```diff
@@ -76,6 +76,14 @@
               {{ $t('simple') }}
             </b-navbar-item>
           </b-tooltip>
+          <b-tooltip
+            label="AI powered process to create your NFT"
+            position="is-right"
+            style="display: block">
```

Comment:
Were you looking for this?
```suggestion
            append-to-body>
```